purchases/src/main/kotlin/com/revenuecat/purchases/coroutinesExtensions.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases
+
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+
+/**
+ * Get latest available customer info.
+ * Coroutine friendly version of [Purchases.getCustomerInfo].
+ *
+ * @return The [CustomerInfo] or a [PurchasesException] with the [PurchasesError]
```

Comment:
I believe there is a `@throws` we can add to the documentation to make it clearer. We should probably add it for better documentation.